lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -1,18 +1,71 @@
 # -*- coding: utf-8 -*-
-
 # Copyright: Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
 
-# NOTE: this file is here to allow modules using the new attributes feature to
-#       work w/o errors in this version of ansible, it does NOT provide the full
-#       attributes feature, just a shim to avoid the fragment not being found.
-
 class ModuleDocFragment(object):
 
     # Standard documentation fragment
     DOCUMENTATION = r'''
-options: {}
```

Comment:
Not having `options: {}` makes it impossible to use this docs fragment with ansible-core 2.11 (or any version before), since that version explicitly checks for `options` being present and will error out if it does not: https://github.com/ansible/ansible/blob/070ae15c522ae2eb7481cb9ef1cc20b8a49b4f5c/lib/ansible/utils/plugin_docs.py#L182-L183